Server/JokesOnYou.Web.Api/Repositories/UserRepository.cs

```diff
@@ -35,16 +50,64 @@ public async Task<User> CreateUserAsync(UserRegisterDTO userRegisterDTO)
                     message.AppendLine(error.Description);
                 }
 
-                throw new UserRegisterException(message.ToString());
+                throw new UserRegisterException(message.ToString(), new ArgumentException());
```

Comment:
why create a new ArgumentException here ?
what information does that add?

Reply:
It's an innerexception, i have no clue as to what it does, but Chirs suggested i add it.

Reply:
If we want to use innerexceptions then this is how you do it. An innerexception is the actual reason the exception occured in the first place, and this was a argument exception. It will show up in the console log, thats pretty much all it does.

Reply:
Yes I cant figure out how to get exceptions out of the errors.
so this is good.